
 This patch resolves a command parser issue 
 causing a director segfault when using something
 like "run job 1 2"

 Apply to version 2.2.4 (and perhaps older 2.2.x versions) with

 cd <bacula-source>
 patch -p0 <2.2.4-parse-command.patch
 ./configure (your options)
 make
 ...
 make install


Index: src/dird/ua_run.c
===================================================================
--- src/dird/ua_run.c	(révision 5616)
+++ src/dird/ua_run.c	(copie de travail)
@@ -933,7 +933,7 @@
             /* Note, yes and run have no value, so do not fail */
             if (!ua->argv[i] && j != YES_POS /*yes*/) {
                ua->send_msg(_("Value missing for keyword %s\n"), ua->argk[i]);
-               return true;
+               return false;
             }
             Dmsg1(800, "Got keyword=%s\n", NPRT(kw[j]));
             switch (j) {
